pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTransactions.java

```diff
@@ -81,10 +81,25 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Get transaction status")
+    private class GetTransactionStatus extends CliCommand {
+        @Parameter(names = {"-m", "--most-sig-bits"}, description = "the most sig bits", required = true)
+        private int mostSigBits;
+
+        @Parameter(names = {"-l", "--least-sig-bits"}, description = "the least sig bits", required = true)
+        private long leastSigBits;
+
+        @Override
+        void run() throws Exception {
+            print(getAdmin().transactions().getTransactionStatus(new TxnID(mostSigBits, leastSigBits)));
+        }
+    }
+
     public CmdTransactions(Supplier<PulsarAdmin> admin) {
         super("transactions", admin);
         jcommander.addCommand("coordinator-status", new GetCoordinatorStatus());
         jcommander.addCommand("transaction-in-buffer-stats", new GetTransactionInBufferStats());
         jcommander.addCommand("transaction-in-pending-ack-stats", new GetTransactionInPendingAckStats());
+        jcommander.addCommand("transaction-status", new GetTransactionStatus());
```

Comment:
It's better to use `transaction-metadata` here since we are using a transaction ID to fetch all transaction metadata through this command.